docs/development/deployment.md

```diff
@@ -60,6 +61,204 @@ If you want to reset your environment you can run the `hack/util-update-app-of-a
 
 Note running these scripts in a clone repo will have no effect as the repo will remain `https://github.com/redhat-appstudio/infra-deployments.git`
 
+### `preview.env` instructions
+
+I want a development deployment of StoneSoup where I can:
```

Comment:
Wasn't sure if I should call it StoneSoup, AppStudio or RHTAP. All 3 are used in some places here, StoneSoup and RHTAP with about equal frequency, AppStudio not so much

Reply:
The latest name is RHTAP but this documentation uses Stonesoup so for consistency within this document, I would use Stonesoup and we should do a separate PR if we want to rename Stonesoup to RHTAP